### PR TITLE
feature/challange-form-fix-1604

### PR DESCRIPTION
**Pull request by rabiul042**
solved challenge form validation issue both existing user and email.
Fix https://github.com/Plant-for-the-Planet-org/treecounter-app/issues/1604

### CODE REVIEW

File: app/locales/en/challengelabels.json [hunk 1]
```diff
@@ -19,5 +19,7 @@
   "challenge_create_successfully": "Challenge created successfully",
   "please_select_user": "Please select user",
   "please_select_year": "Please select year",
-  "could_not_add_user": "Could not add user"
+  "could_not_add_user": "Could not add user",
+  "name_is_required": "Name is Required\n",
+  "goal_is_required": "Challenge is Required\n"
```
Comment (mahmed0715) [inline app/locales/en/challengelabels.json]: Same thing need in de folder(German)

Comment (norbertschuler) [reply app/locales/en/challengelabels.json]: The German resources will be added with a translation workflow once run in develop branch, so adding it within a pull request for a new feature isn't really necessary: https://github.com/Plant-for-the-Planet-org/treecounter-app/wiki/Translation-workflow

**Comment by mahmed0715** [comment]
@norbertschuler please test this PR.

**Comment by mahmed0715** [comment]
Yeah got it, @rabiul042 is working to prevent form being submitted to backend while there is issue with form. @norbertschuler

**Pull request merged**